eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -73,7 +73,7 @@ public function loadBinaryFile($binaryFileId)
     {
         $this->checkBinaryFileId($binaryFileId);
 
-        if ($binaryFileId[0] === '/') {
+        if ($binaryFileId[0] === '/' || (PHP_OS === 'WINNT' && $binaryFileId[1] === ':')) {
```

Comment:
@bdunogier: I looked into it once again. Here is the data that causes this to fail:
https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/SPI/Tests/FieldType/ImageIntegrationTest.php#L307

For the code:
`'id' => ($path = __DIR__ . '/_fixtures/image.jpg')`
`__DIR__` constant is always resolved as an absolute path, so on Linux it will start with `/` and on Windows with e.g.: `C:\`.

The goal of this test is to fail: [ImageIntegrationTest::testCreateContentUsingIdPropertyThrowsWarning](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/SPI/Tests/FieldType/ImageIntegrationTest.php#L285)
But the problem is it fails with an incorrect exception thrown due this: `$binaryFileId[0] === '/')` being false on Windows.